roles/ood/tasks/main.yaml

```diff
@@ -46,3 +46,7 @@
 
 - name: Set SELinux to permissive mode without reboot
   command: setenforce 0
+
+- name: scp the slurm config
+  shell: scp ohpc:/etc/slurm/slurm.conf /etc/slurm/slurm.conf
```

Comment:
This task is still dependent on ssh credentials between the accounts, host name resolution, and static node names.
 
Can you propose a change that would copy the file between the nodes by way of vagrant?  This would need to be in the ohpc_vagrant repo.

Reply:
Vagrant-scp is a plugin which copies the files from host to guest VM.
User needs to install the plugin via `vagrant plugin install vagrant-scp` 

From what I understand 
`vagrant scp [vm_name]:<somewhere_on_the_vm> <some_local_file_or_dir>` could copy the file to the host and place it in the shared directory. After that file provisioner in vagrant can allow you to upload a file from the host machine to the guest machine while doing `vagrant up ood`.

Vagrant.configure("2") do |config|
  config.vm.provision "file", source: "path/to/file", destination: "path/to/file"
end

Reply:
Regarding the conf.vm.provision line, this would make sense only after the ansible scripts have completed.

It would need to run after the config.vm.provision "shell" line.  Please propose this solution in the context of a feature and merge request on our ohpc_vagrant project.

Reply:
Another way Ravi proposed while we were discussing this is to generate the ssh keys and distribute them to the VMs in the cluster so that all of them will have the same key and we can ssh and scp over to any machine.